Dockerfile

```
@@ -45,30 +45,13 @@ RUN apk add --no-cache ruby \
   "asciidoctor-pdf:${ASCIIDOCTOR_PDF_VERSION}"
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-# Haskell build for: erd
+# Install erd-go (https://github.com/kaishuu0123/erd-go) as replacement for erd (https://github.com/BurntSushi/erd)
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-# Alpine 3.16 is the last known working version for building erd (dynamic linking)
-FROM alpine:3.16 AS erd-builder
+FROM golang:alpine as erd-builder
+RUN apk add git \
+    && go install github.com/kaishuu0123/erd-go@v2.0.0
```

This version line shows the following error:

```
github.com/kaishuu0123/erd-go@v2.0.0: github.com/kaishuu0123/erd-go@v2.0.0: invalid version: module contains a go.mod file, so module path must match major version ("github.com/kaishuu0123/erd-go/v2")
```

```suggestion
    && go install github.com/kaishuu0123/erd-go/v2@v2.0.0
```